pom.xml

```diff
@@ -99,6 +99,13 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
 
+        <!-- Module names used for Java 9 modules and OSGi bundles -->
+        <hibernate-validator.module-name>org.hibernate.validator</hibernate-validator.module-name>
+        <hibernate-validator-cdi.module-name>org.hibernate.validator.cdi</hibernate-validator-cdi.module-name>
+        <!-- TODO in 6.1: make the names consistent by removing the bundle properties, see HV-1484 -->
+        <hibernate-validator.bundle-name>org.hibernate.validator.hibernate-validator</hibernate-validator.bundle-name>
+        <hibernate-validator-cdi.bundle-name>org.hibernate.validator.hibernate-validator-cdi</hibernate-validator-cdi.bundle-name>
```

Comment:
We already could use the right one for the CDI module already, no?

Reply:
Why? It already had a name too.

Reply:
The thing is that if we don't explicitly define one, a name is forged from the Maven coordinates. So even, if not specified, each bundle already has a name. It's just that, now, the name is specified and clear. And we will change it in 6.1.

Reply:
Makes sense.